<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1673</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="player1ProgressBar_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="player2ProgressBar_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="gameInfoLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>1091</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Game ongoing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switchPlayerButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Switch player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="player1Time">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>390</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Pelaajan 1 J채ljell채 oleva aika</string>
    </property>
   </widget>
   <widget class="QPushButton" name="player2Time">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Pelaajan 2 j채ljell채 oleva aika</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startGameButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>610</y>
      <width>471</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START GAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopGameButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>610</y>
      <width>491</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>STOP GAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="player1TimeLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>470</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="player2TimeLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>470</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1673</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
